File: SDKs/amazonaws.com/fms/2018-01-01/java/lib/src/main/java/openapisdk/models/shared/Ec2CreateRouteTableAction.java
package openapisdk.models.shared;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonInclude.Include;

/**
 * Ec2CreateRouteTableAction
 * Information about the CreateRouteTable action in Amazon EC2.
**/
public class Ec2CreateRouteTableAction {
    @JsonInclude(Include.NON_ABSENT)
    @JsonProperty("Description")
    public String description;
    public Ec2CreateRouteTableAction withDescription(String description) {
        this.description = description;
        return this;
    }
    @JsonProperty("VpcId")
    public ActionTarget vpcId;
    public Ec2CreateRouteTableAction withVpcId(ActionTarget vpcId) {
        this.vpcId = vpcId;
        return this;
    }
}

File: SDKs/amazonaws.com/fms/2018-01-01/python/src/sdk/models/shared/ec2createroutetableaction.py
import dataclasses
from typing import Optional
from dataclasses_json import dataclass_json
from sdk import utils
from ..shared import actiontarget as shared_actiontarget


@dataclass_json
@dataclasses.dataclass
class Ec2CreateRouteTableAction:
    r"""Ec2CreateRouteTableAction
    Information about the CreateRouteTable action in Amazon EC2.
    """
    
    vpc_id: shared_actiontarget.ActionTarget = dataclasses.field(metadata={'dataclasses_json': { 'letter_case': utils.field_name('VpcId') }})
    description: Optional[str] = dataclasses.field(default=None, metadata={'dataclasses_json': { 'letter_case': utils.field_name('Description') }})
    
